lib/simulation/openfisca/index.js

```diff
@@ -1,26 +1,56 @@
 /*
 ** Module dependencies
 */
-var request = require('superagent');
-var util = require('util');
 var config = require('../../config/config');
+var http = require('http');
 var mapping = require('./mapping');
 var reverseMap = require('./mapping/reverse');
+var url = require('url');
+var util = require('util');
+
+var openfiscaURL = url.parse(config.openfiscaApi);
 
 function calculate(situation, debug, callback) {
-    request
-        .post(config.openfiscaApi + '/api/1/calculate')
-        .send(mapping.buildOpenFiscaRequest(situation, debug))
-        .end(function(err, response) {
-            if (err) return callback(err);
+    var openfiscaRequest = mapping.buildOpenFiscaRequest(situation, debug);
+    openfiscaRequest.scenarios[0].test_case.familles[0].aspah = openfiscaRequest.scenarios[0].test_case.familles[0].aspa;
```

Comment:
Thanks, you've been too fast in your review. I pushed the commits to make sure they were on GH but I haven't done a review before requesting another review. Thanks.